ibis/expr/api.py

```diff
@@ -26,77 +26,13 @@
 from ibis.expr.deferred import Deferred
 from ibis.expr.schema import Schema
 from ibis.expr.sql import parse_sql, show_sql, to_sql
-from ibis.expr.types import (  # noqa: F401
-    ArrayColumn,
```

Comment:
This looks like a breaking change. Are you sure that we should remove these?

Reply:
It is, but these types are stable under `ibis.expr.types as ir`. I'd rather have just one place to access them than a hodgepodge list of things that doesn't actually include everything and has to be manually maintained.

Reply:
Can you separate this into another commit including the breaking change note?

Reply:
Will do!

Reply:
Done.